.buildkite/x-pack/pipeline.xpack.libbeat.yml

```diff
@@ -18,45 +14,176 @@ env:
+    steps:
+      - label: ":windows: Windows 10 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/libbeat
+          mage -w reader\etw build goUnitTest
+        key: "extended-win-10-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_10}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Windows 10 Unit Tests"
+
+      - label: ":windows: Windows 11 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/libbeat
+          mage -w reader\etw build goUnitTest
+        key: "extended-win-11-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_11}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Windows 11 Unit Tests"
+
+      - label: ":windows: Windows 2019 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/libbeat
+          mage -w reader\etw build goUnitTest
+        key: "extended-win-2019-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2019}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Windows 2019 Unit Tests"
 
-  - input: "Input Parameters"
-    key: "input-run-all-stages"
-    fields:
-    - select: "Libbeat - run_xpack_libbeat"
-      key: "run_xpack_libbeat"
-      options:
-        - label: "True"
-          value: "true"
-        - label: "False"
-          value: "false"
-      default: "false"
-    - select: "Libbeat - run_xpack_libbeat_arm_tests"
-      key: "run_xpack_libbeat_arm_tests"
-      options:
-        - label: "True"
-          value: "true"
-        - label: "False"
-          value: "false"
-      default: "false"
-    if: "build.source == 'ui'"
-
-  - wait: ~
-    if: "build.source == 'ui'"
-    allow_dependency_failure: false
-
-  - label: ":linux: Load dynamic x-pack libbeat pipeline"
-    key: "libbeat-pipeline"
-    command: ".buildkite/scripts/generate_xpack_libbeat_pipeline.sh"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci:latest"
-    notify:
-      - github_commit_status:
-          context: "${BEATS_PROJECT_NAME}: Load dynamic pipeline's steps"
+  - group: "Extended Tests"
```

Comment:
can we change to `x-pack/libbeat Linux arm Extended Tests` for consistency as done, e.g. in https://github.com/elastic/beats/pull/38828/files#diff-4291f1a0414f4870861926dce4a866be0c14ce2b86f018e958458c53c719c7a2R196